### ISSUE TITLE
Stop signs will Stack while Street Signs will not

### ISSUE THREAD

**Issue by Conchipedia** [StrangeLoopGames/EcoIssues#18753]
**Version:** 0.9.0.5 beta release-100

i made 10 street sign and 10 stop signs, the street signs refuse to stack while the stops signs will
![snap7428](https://user-images.githubusercontent.com/29383891/95932353-8e2c8c00-0dc3-11eb-98d7-3d00fd14956d.png)

**Comment by SlayksWood** [comment]
This happens because Street sign has text component and it should save it for each sign. So you can't stack them.
@olegshegai Nothing can be changed for signs without text?

**Comment by olegshegai** [comment]
Yeah, good idea, but it's currently possible with some unwanted workarounds.
We may add some rule that will allow stacking objects with no persistent data if they got a specific tag or attribute, but that require some time. I suggest to make it not-high-priority feature.

**Comment by Conchipedia** [comment]
looks like ALL crafted signs won;t stack either as it saves the writing
is this really a necessity that we keep the writing saved?
they take up way too much space just sitting 16 to a chest or 25 to stockpile rather than stacking
